Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -133,7 +138,8 @@ def row_centery(row):
     return c
 
 class Table:
-    re_month = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
+    re_month = re.compile("(january|february|march|april|may|june|july|august\
+                          |september|october|november|december)")

```

Comment:
@Evildoor,

> I can't say that i'm thrilled with all this history rewriting for the sake of perfection. Especially when it requires additional work. This does not refers to the previous point, which i approve.

The repository itself is an additional work ;)

Why do I think it is better to join these two commits:
1. Functionality.
Every commit is a fixed state of the repository. It is better to have a **working state**, then a broken one, so that one can checkout the repository at every point and check how\`s it going.
However here we have 4 repository states in a row where the code can\`t be compiled, _and we have a possibility to fix it._
Yes, this is a development branch, and since we use the **merge** policy (as a contrary to **squash** or fast-forward **rebase and merge**) it means in the "general" branch we still have the working state.
But, by default, when you watch the commit history (say, via the GitHub web-interface, or with `git log`) you still see the linear history with all the commits from the development branch alongside with all the others, so why threat them differently?

2. Readability.
As it is said above, the development commints are not "hidden" behind the merge commit, they are treated as a valuable points of the history. Thus it looks like a good idea to have it more compact, not splitting one logical change ("package import style changed") into a number of commits with the same idea ("import style changed", "import style changed pt.2: fix pt.1", ...)
It gives no new information (except that the first commit is not a thing-in-itself and should not be used individually for any purposes), yet distracts attention from the main course of changes, making one to re-read history backwards to understand which commit was fixed.

I mean **yes, it is a perfectionism. But it is not only for better looks.**